src/flightlog_index.js

```diff
@@ -140,9 +142,9 @@ export function FlightLogIndex(logData) {
           switch (frameType) {
             case "P":
             case "I":
-              var frameTime =
+              let frameTime =
                 frame[FlightLogParser.prototype.FLIGHT_LOG_FIELD_INDEX_TIME];
```

Comment:
Would perfectly fit on a single line :)
```suggestion
              let frameTime = frame[FlightLogParser.prototype.FLIGHT_LOG_FIELD_INDEX_TIME];
```

Reply:
Yes, ofcourse. Improwed now